lib/scala/syntax/specialization/shared/src/test/scala/org/enso/syntax/text/DocParserTests.scala

```diff
@@ -643,9 +643,15 @@ class DocParserTests extends AnyFlatSpec with Matchers {
     ),
     Synopsis(Section.Raw("Foo"))
   )
-  """   ALAMAKOTA a kot ma ale
+  """   ALAMAKOTA
+    | DEPRECATED
+    | PRIVATE
     | foo bar""".stripMargin.replaceAll(System.lineSeparator(), "\n") ?= Doc(
-    Tags(Tags.Tag(3, Tags.Tag.Unrecognized, "ALAMAKOTA a kot ma ale")),
+    Tags(
+      Tags.Tag(3, Tags.Tag.Unrecognized, "ALAMAKOTA"),
+      Tags.Tag(1, Tags.Tag.Type.Deprecated),
+      Tags.Tag(1, Tags.Tag.Type.Private)
```

Comment:
Please add some suffix text to these tags like the specification says is supported.